tests/basic_correctness/test_basic_correctness.py

```diff
@@ -3,16 +3,29 @@
 Run `pytest tests/basic_correctness/test_basic_correctness.py`.
 """
 import os
+import weakref
 
 import pytest
 
+from vllm import LLM
+
 MODELS = [
     "facebook/opt-125m",
     "meta-llama/Llama-2-7b-hf",
 ]
 VLLM_ATTENTION_BACKEND = "VLLM_ATTENTION_BACKEND"
 
 
+def test_vllm_gc_ed():
+    """Verify vllm instance is GC'ed when it is deleted"""
```

Comment:
NOTE: This passes without this fix in the master because tensorizer is always downloaded.

Is it easy to add any kind of "minimal dependency test"?

Reply:
Maybe can just run `pip uninstall` before the test?

Reply:
Hmm this can have trouble if tests fail iiuc (because we don't have "fixture" kind of thing in pipeline yaml)

Reply:
but looks like it is kind of already used. One thing I can think of is to not include tensorizer dep by default and install it before running a tensorizer test. wdyt? @simon-mo @DarkLight1337